ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_tensor.py

```diff
@@ -1448,3 +1448,32 @@ def test_paddle_greater_than(
         frontend=frontend,
         on_device=on_device,
     )
+
+
+# floor_divide
+def test_paddle_floor_divide(
+    dtypes_and_x,
```

Comment:
I think you've forgotten to use the `@frontend_method` decorator here. You can look at the other examples above to see how to correctly write the test function 🙂